### PR TITLE
Merging DMP concepts from WebProtégé

### PR DESCRIPTION
**Pull request by matuskalas**

### CODE REVIEW

File: EDAM_dev.owl [hunk 1]
```diff
@@ -37423,7 +37449,41 @@
...
         <rdfs:label>MSP</rdfs:label>
     </owl:Class>
-	
+    
+
+
+    <!-- http://edamontology.org/format_4041 -->
+
+    <owl:Class rdf:about="http://edamontology.org/format_4041">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3464"/>"http://www.w3.org/2002/07/owl#DeprecatedClass"/>
```
Comment (matuskalas) [inline EDAM_dev.owl:37458]:
```suggestion
        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3464"/>
```

Comment (matuskalas) [inline EDAM_dev.owl:37458]:
```suggestion
        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3464"/>
```

Comment (matuskalas) [reply EDAM_dev.owl:37458]: WTF, it doesn't work to commit this :( (GH askes me in a loop to refresh and try again)

Comment (matuskalas) [reply EDAM_dev.owl:37458]: Fixed locally & pushed, closing this comment

**Review by matuskalas — COMMENTED** [review]
Should we merge this, before moving on to operations? @bianchini88

**Review by bianchini88 — APPROVED** [review]

**Comment by bianchini88** [comment]
@matuskalas Feel free to merge. (Note that I am not authorised to merge PRs, which is probably not a bad thing at this stage).

**Comment by matuskalas** [comment]
> @matuskalas Feel free to merge. (Note that I am not authorised to merge PRs, which is probably not a bad thing at this stage).

Million thanks @bianchini88 🙏🏽🙏🏽 Alright, so with the "Write" permissions you can push and approve, but not merge to main, it looks like.

**Pull request merged**